stderr of test 'mosaic_oid_with_nils.dict256` in directory 'monetdb5/modules/mosaic` itself:


# 13:50:20 >  
# 13:50:20 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=38679" "--set" "mapi_usock=/var/tmp/mtest-16251/.s.monetdb.38679" "--forcemito" "--dbpath=/home/aris/sources/monetdb/build/installation/var/MonetDB/mTests_monetdb5_modules_mosaic" "--set" "embedded_c=true"
# 13:50:20 >  

# builtin opt 	gdk_dbpath = /home/aris/sources/monetdb/build/installation/var/monetdb5/dbfarm/demo
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_ipv6 = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 38679
# cmdline opt 	mapi_usock = /var/tmp/mtest-16251/.s.monetdb.38679
# cmdline opt 	gdk_dbpath = /home/aris/sources/monetdb/build/installation/var/MonetDB/mTests_monetdb5_modules_mosaic
# cmdline opt 	embedded_c = true

# 13:50:20 >  
# 13:50:20 >  "/usr/bin/python3" "mosaic_oid_1.frame.MAL.py" "mosaic_oid_1.frame"
# 13:50:20 >  


# 13:50:21 >  
# 13:50:21 >  "Done."
# 13:50:21 >  

